clmm/theory/ccl.py

```diff
@@ -152,5 +152,52 @@ def _eval_excess_surface_density(self, r_proj, z_cl):
         else:
             return self.eval_mean_surface_density(r_proj, z_cl) - self.eval_surface_density(r_proj, z_cl)
 
-
+    def _prepare_ccl(self, r_proj, z_cl, z_src):
```

Comment:
I created this function to prepare the objects for the ccl functions. Since we want to be able to give a list of projected radius and source redshifts, this object turns the given data into lists, even if there is only one data for each variable.

Also, the ccl uses the scale factor instead of the redshift, so I had to compute the variables. Lastly, I use np.vectorize so this functions works the same way as the NumCosmo backend.